config/api.yml

```diff
@@ -2779,6 +2779,8 @@
         :identifier: rbac_user_edit
       - :name: delete
         :identifier: rbac_user_delete
+      - :name: set_current_group
+        :identifier: rbac_user_edit
```

Comment:
does this mean if a user does not have that role they cannot change their current group ?

Reply:
@abellotti yeah, I guess so. I am not sure if users should always be allowed to change their own group or if they need that particular role?

Reply:
May be easy to verify, in classic UI login as a user without that role but a member of a couple of groups, see if you can change the current group in the upper right hand pull-down. If you can, we'll need to somehow handle that.

Reply:
@abellotti verified that a user in the classic UI without this edit role is still able to change their group. updated accordingly 👍